components/recipe/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import Image from 'next/image'
+import styles from './Recipe.module.scss'
+import Chicken from '../../assets/Chicken.jpg'
+import Steak from '../../assets/Steak.jpg'
+import Fish from '../../assets/Fish.jpg'
```

Comment:
We should create a utility file that imports all 3 of these images and exports them all as an object of variables. 

Folder/file:
Utilities/recipeImages.js

```suggestion
import Fish from '../../assets/Fish.jpg'
import Chicken from "../../assets/Chicken.jpg"

export {
 Fish,
 Chicken
}
```

Reply:
Then import the 3 images into this file like this:

import { Fish, Chicken } from "folder"